sql/moz-fx-data-shared-prod/search_terms_derived/adm_weekly_aggregates_v1/query.sql

```diff
@@ -0,0 +1,20 @@
+SELECT
+  search_query AS query,
+  block_id,
+  position,
```

Comment:
There are some strange values for position (i.e., values much greater than 10). If we don't clean this up, we'll likely get questions about it from adM via Aaron. @teonbrooks , have you found reasons why there can be urlbar positions > 10? 

My preference would be to send adM the raw data (unless they explicitly asked for something else), but have some answers on hand for when they ask

Reply:
Per discussion with AKim, we will remove position.